crypto/src/main/kotlin/net/corda/v5/crypto/PublicKeyHash.kt

```diff
@@ -1,7 +1,7 @@
-@file:JvmName("PublicKeyUtils")
 package net.corda.v5.crypto
```

Comment:
This is still a Kotlin file. And is this where `KeyUtils.java` came from? It looks like its original name was actually `PublicKeyUtils`.

There cannot be _any_ Kotlin references left in `main`.

Reply:
Ported to Java now (since I'm continuning to work on the branch)